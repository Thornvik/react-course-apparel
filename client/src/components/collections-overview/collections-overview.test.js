import React from "react";
import { shallow } from "enzyme";
import { CollectionsOverview } from "./collections-overview.component";

it("should render Collections-Overview component", () => {
  expect(shallow(<CollectionsOverview collections={[]} />)).toMatchSnapshot();
});
