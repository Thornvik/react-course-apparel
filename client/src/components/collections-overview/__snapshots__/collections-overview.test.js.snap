// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Collections-Overview component 1`] = `ShallowWrapper {}`;
